edx_exams/apps/api/v1/tests/test_views.py

```diff
@@ -219,10 +219,8 @@ def test_exam_modified_type(self):
         self.assertEqual(proctored_exam.is_active, True)
 
     @ddt.data(
-        (True, 'proctored', True),  # test case for a proctored exam with no course config
         (False, 'proctored', False),  # test case for a proctored exam with a course config
         (False, 'timed', True),  # test case for a timed exam with a course config
-        (True, 'timed', True)  # test case for a timed exam with no course config
```

Comment:
at the moment, if there's no course config these requests are redirected to edx-proctoring.

Reply:
Question: could you clarify what the first True in the tuple signifies? Thanks!

Reply:
it signifies using a different course id that doesn't have a configuration in edx-exams so I removed these cases for the time being since so long as the new router app exists those requests will never hit this view